playbook.md

```diff
@@ -30,7 +30,7 @@ We think that it's very important to have a talented team if we're going to succ
 We are helpful, useful and expert. We give practical and pragmatic advice to each other, and to our clients.
 
 #### Honest
-We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or obfuscate.
+We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or wilfully confuse.
```

Comment:
I feel like the original sentence used obfuscate in the sense of _hiding_ things from the client. 

The point is that while actively deceiving/misleading/confusing the client is obviously wrong, hiding things from the client (e.g. not telling them the full detail of a situation) might be considered pragmatic, so this sentence reinforces it as something we decide to not do.